documentation/modules/mtv-shared-disks.adoc

```diff
@@ -0,0 +1,224 @@
+ migrateSharedDisks: false
+ provider:
+   destination:
+     apiVersion: forklift.konveyor.io/v1beta1
+     kind: Provider
+     name: host
+     namespace: openshift-mtv
+     uid: abf4509f-1d5f-4ff6-b1f2-18206136922a
+   source:
+     apiVersion: forklift.konveyor.io/v1beta1
+     kind: Provider
+     name: vsphere
+     namespace: openshift-mtv
+     uid: be4dc7ab-fedd-460a-acae-a850f6b9543f
+ targetNamespace: openshift-mtv
+ vms:
+   - id: vm-71
+     name: vm-2-with-shared-disks
+----
+
+. Start the migration of the second plan and wait for it to finish.
+. Verify that all shared disks are attached to the same VMs as they were before migration and that none are duplicated. In case of problems, see the discussion of known issues that follows. 
+
+[id="known-issues-shared-disks_{context}"]
+== Known issues
+
+[id="cyclic-shared-disk_{context}"]
+=== Cyclic shared disk dependencies
+
+Problem: VMs with cyclic shared disk dependencies cannot be migrated successfully.
+
+Explanation: When `migrateSharedDisks` is set to `true`, {project-short} migrates each VM in the plan, one by one, and any shared disks attached to it, without determining if a shared disk was already migrated. 
+
+In the case of 2 VMs sharing one disk, there is no problem. {project-short} transfers the shared disk and attaches the 2 VMs to the shared disk after the migration. 
+
+However, if there is a cyclic dependency of shared disks between 3 or more VMs, {project-short} either duplicates or omits one of the shared disks. The figure that follows illustrates the simplest version of this problem.
+
+.Simple example of cyclic shared disks
+image::simple-cyclic-shared-disks.png[Simple cyclic shared disk dependencies]
+
+In this case, the VMs and shared disks cannot be migrated in the same `Plan` CR. Although this problem could be solved using `migrateSharedDisks` and 2 `Plan` CRs, it illustrates the basic issue that must be avoided in migrating VMs with shared disks.
+
+[id="cyclic-shared-disk-workarounds_{context}"]
+=== Workarounds
+
+As discussed previously, it is important to try to create 2 `Plan` CRs in which each shared disk is migrated once. However, if your migration does result in a shared disk either duplicated or not being transferred, you can use one of the following workarounds:
+
+* Duplicate one of the shared disks
+* "Remove" one of the shared disks
+
+[id="duplicate-shared-disk_{context}"]
+==== Duplicate a shared disk
+
+In the figure that follows, VMs 2 and 3 are migrated with the shared disks in the first plan, and VM 1 is migrated in the second plan. Doing this breaks the cyclic dependencies, but this workaround has a drawback: It results in shared disk 3 being duplicated. The solution is to remove the duplicated PV and migrate VM 1 again.
+
+.Duplicated shared disk
+image::cyclic_workaround1.png[Duplicate a shared disk]
+
+Advantage:
+
+The source VMs are not affected.
+
+Disadvantage:
+
+One shared disk gets transferred twice.
```

Comment:
`One shared disk gets transferred twice, and you need to manually delete the duplicate disk and reconnect VM 3 to shared disk 3 in Red Hat OpenShift after the migration.